Sources/Features/Client/BacktraceReporter.swift

```diff
@@ -6,8 +6,9 @@ final class BacktraceReporter {
     private(set) var api: BacktraceApi
     private let watcher: BacktraceWatcher<PersistentRepository<BacktraceReport>>
     private(set) var attributesProvider: SignalContext
-    private(set) var backtraceOomWatcher: BacktraceOomWatcher
+    private(set) var backtraceOomWatcher: BacktraceOomWatcher!
     let repository: PersistentRepository<BacktraceReport>
+    var oomSupport : Bool = true
```

Comment:
Should it be inited with true? Or `self.configuration.detectOom`? Or maybe we can avoid this var altogether.